src/items/_tray.scss

```diff
@@ -44,7 +44,7 @@
 }
 
 .osjs-root[data-dir="rtl"] {
-  .osjs-panel-item[data-name=tray] > div:not(:last-child) {
+  .osjs-panel-item[data-name=tray] > * > div:not(:last-child) {
```

Comment:
Looks like you missed an asterisk here bud :)

Reply:
Oops! How did that happen? 😭